drivers/sensor/vl53l1x/vl53l1_platform_user_data.h

```diff
@@ -0,0 +1,97 @@
+
+/*
```

Comment:
Not sure all these files are actually needed here

Reply:
they're included in the platform-folder of STSW-IMG007 and basically I just copied everything from there to the driver folder - I think it makes sense.
I just wonder why the driver got so much more files compared to VL53L0X.
Also, atm 
VL53L1_WriteMulti
is not working (as far as I can tell the other I2C funcs work).

I'll push the updated files but they will still have most of the style errors.